server/service/Converter.py

```diff
@@ -0,0 +1,53 @@
+import json
+
+from server.service.data.Document import Cell, Row, Table, Document
+
+
+class Converter:
+
+    @staticmethod
+    def convert_to_json(document: Document):
+        result = {}
+        table_num = 1
+
+        for table in document.tables:
+            t = {}
+            row_num = 1
+            for row in table.rows:
+                r = []
+                for cell in row.cells:
+                    r.append(cell.text)
+                t["row" + str(row_num)] = r
```

Comment:
Czy to jest dobre podejście? Klucze słowników mogą przemieszać się i wtedy potrzebowalibyśmy sortować wyniki po kluczach. Może lepiej byłoby po prostu użyć zwykłej listy. Jeśli chcemy przechowywać dodatkowe metadane dla poszczególnych wierszy, to można byłoby zrobić listę słowników, gdzie każdy słownik ma jakieś dodatkowe pola i pole np. data, gdzie trzyma sam wiersz